assets/js/team.tsx

```diff
@@ -67,42 +71,66 @@ function AlumRoster({ members }: { members: Alum[] }) {
   );
 }
 
-export function loadTeam() {
-  const currentUser = loggedInUser();
+function Rosters() {
+  const nav = document.getElementById("sidenav-wrapper") as HTMLElement;
```

Comment:
Not sure I'd typically expect a component to have such awareness of DOM details like this. Might expect more like:

```suggestion
function Rosters({ nav }: { nav: HTMLElement }) {
```

Or:

```suggestion
function Rosters({ navSelector }: { navSelector: string }) {
  const nav = document.querySelector(navSelector) as HTMLElement;
```